index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio-Jorum Odiemo</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header class="Top">
+            <a href=""><img src="images/MyLogo.png" alt="My Logo" ></a>
+            <button type="button"><img class="menu" src="images/menu.png" alt="Menu"></button>           
+    </header>
+         <main>
+         <section class="headline">
+            <div class="headline-box">
+              <h2 class="mainheader">
+                  I’m Jorum<br />Glad to see you!</h2>
```

Comment:
- [ ]  Please replace `<h2>` with the `<h1>` tag because the `<h1>` tag stands for the most important heading of the web page and the `<h2>`...`<h6>` stands for the least important and smallest one..